app/services/update_user.rb

```diff
@@ -33,7 +33,24 @@ def update_sam
     reckoner.delay.set!
   end
 
+  def parsed_user_params
+    user_params.each_with_object({ }) do |(key, value), hash|
+      converted_value = nil_not_empty_string(value)
+      hash[key] = converted_value
+    end
+  end
```

Comment:
Yeah, I think I've just deleted the blank params in the past when I've had this issue. But that would make for a weird UX issue if a user cleared their name. Instead of getting a validation error (for the name being set to `nil`), it would just not update the name because the param was removed.